js/moderation.ts

```diff
@@ -0,0 +1,66 @@
+import { Scorer } from "@braintrust/core";
```

Comment:
make sure to export it from `index.ts`

Reply:
Added: https://github.com/braintrustdata/autoevals/pull/63/commits/10a36d9785a7428d7c8db3a9385367d10282dae6#diff-8e7ddf1b04e664328fd86034457d7b5297c4b3e35bc68fa091b215f9011de40aR34